src/style.css

```
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  background: #fff;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+}
+
+.input {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
+  width: 100%;
+  height: 70px;
+  display: block;
+  margin-top: 20px;
+  background: #fff ;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  padding-left: 62px;
+  border-radius: 4px;
+  outline: none;
+}
+
+.input::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.smallBar {
+  position: relative;
+}
+
+.input--small {
+  width: 100%;
+  height: 42px;
+  display: block;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  padding-left: 33px;
+  border-radius: 4px;
+}
```

Do not repeat your self. You already have those styles in `.input`
```suggestion
.input--small {
  height: 42px;
  padding-left: 33px;
}
```